packages/transition-backend/src/services/transitRouting/TrRoutingBatch.ts

```diff
@@ -258,38 +258,19 @@ class TrRoutingBatch {
         routing: TransitRouting
     ): Promise<{ input: string; csv?: string; detailedCsv?: string; geojson?: string }> => {
         const { resultHandler, pathCollection } = await this.prepareResultData(routing);
-
-        let currentResultPage = 0;
-        let totalCount = 0;
-        const pageSize = 250;
-        do {
-            const { totalCount: total, tripResults } = await resultsDbQueries.collection(this.options.jobId, {
-                pageIndex: currentResultPage,
-                pageSize
+        const resultStream = resultsDbQueries.streamResults(this.options.jobId);
+        for await (const row of resultStream) {
+            const result = resultsDbQueries.resultParser(row);
+            const processedResults = await generateFileOutputResults(result.data, routing.attributes.routingModes, {
```

Comment:
This await though, I wish I could get rid of. It's just that the result generation is using a function that requests the walking part of the paths (async request to OSRM). Ideally, this should have been done during calculation and here we just take it, but when we refactor the results, we can improve that. For now, we'll see if it allows to generate the files in the eod instance